### PR TITLE
[HDInsight] Added settings for C#

### PR DESCRIPTION
**Pull request by wawon-msft**
This change is necessary for compatibility with generate.cmd in the .Net repo.

This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [ ] My spec meets the review criteria:
  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/hdinsight/resource-manager/readme.md [hunk 1]
```diff
@@ -71,3 +70,18 @@ Please also specify `--go-sdk-folder=<path to the root directory of your azure-s
...
+
+## C# 
+
+These settings apply only when `--csharp` is specified on the command line.
+Please also specify `--csharp-sdks-folder=<path to "SDKs" directory of your azure-sdk-for-net clone>`.
+
+``` yaml $(csharp)
+csharp:
+  azure-arm: true
+  license-header: MICROSOFT_APACHE_NO_VERSION
```
Comment (lmazuel) [inline specification/hdinsight/resource-manager/readme.md]: CELA recommands MIT, and it's usually what is used here. Random example:
https://github.com/Azure/azure-rest-api-specs/tree/current/specification/network/resource-manager#c

Could you confirm this is an actual choice, and not random?

File: specification/hdinsight/resource-manager/readme.md [hunk 2]
```diff
@@ -71,3 +70,18 @@ Please also specify `--go-sdk-folder=<path to the root directory of your azure-s
...
+
+
+## C# 
+
+These settings apply only when `--csharp` is specified on the command line.
+Please also specify `--csharp-sdks-folder=<path to "SDKs" directory of your azure-sdk-for-net clone>`.
+
+``` yaml $(csharp)
+csharp:
+  azure-arm: true
```
Comment (lmazuel) [inline specification/hdinsight/resource-manager/readme.md]: Just check with team, please put that top level after `openapi-type`. That's true to all language generation, and will avoid duplication.

**Review by lmazuel — CHANGES_REQUESTED** [review]

File: specification/hdinsight/resource-manager/readme.md (hunk 2 above)
Comment (wawon-msft) [reply specification/hdinsight/resource-manager/readme.md]: Sounds good, fixed.

**Review by lmazuel — APPROVED** [review]
LGTM!

**Pull request merged**